locales/en-US/messages.properties

```diff
@@ -48,6 +48,7 @@ problems_donating_thunderbird2=Problems donating? <a href="https://donate.mozill
 donation_notice_2=Contributions go to the Mozilla Foundation, a 501(c)(3) organization based in Mountain View, California, to be used in its discretion for its charitable purposes. They are tax-deductible in the U.S. to the fullest extent permitted by law.
 
 donation_notice_thunderbird=Contributions go to the Mozilla Foundation, a 501(c)(3) organization, to support Mozilla Thunderbird.
```

Comment:
could you remove this string? In general if we keep a string that’s currently not used, I try to add`# Obsolete string` as a localization note so that we don’t forget to remove it later. But here I think we should just remove it